cobol-parser/src/test/scala/za/co/absa/cobrix/cobol/reader/extractors/raw/RawRecordExtractorFactorySpec.scala

```diff
@@ -32,7 +32,7 @@ class RawRecordExtractorFactorySpec extends WordSpec {
 
     "be able to create a record extractor" in {
       val recordExtractor = RawRecordExtractorFactory.createRecordHeaderParser("za.co.absa.cobrix.cobol.reader.extractors.raw.TextRecordExtractor",
-        RawRecordContext(0L, new TestStringStream("AAA111\nBBB222\n"), copybook, ""))
+        RawRecordContext(0L, new TestStringStream("AAA111\nBBB222\n"), copybook, null, null, ""))
```

Comment:
isn't there a better construct than passing in `nulls` into the api? :) 

e.g. maybe varargs?

Reply:
Yeah, I can use a factory here